### DISCUSSION TITLE
Dropdown right mobile size are not working

### DISCUSSION THREAD

**Discussion by jennafauconnier** [saadeghi/daisyui#2922] [Q&A] [answered]
Hello, i did an example here : https://play.tailwindcss.com/cdvFtSipMF?size=514x720
When i use dropdown right it's not working. It's due to the upgrade from version 2 to the latest 

Does anyone have an idea ? 

Thanks by advance 

Jenna

**Comment by saadeghi** [comment]
I think your link is not correct

**Reply by jennafauconnier** [reply, marked as answer]
My bad, hope this one is good : https://play.tailwindcss.com/cdvFtSipMF?size=514x720
